### DISCUSSION TITLE
Standard Runtime Support

### DISCUSSION THREAD

**Discussion by batesbw** [eschweitzer78/gps-design-systems-lwc#383] [General]
Hi Emmanuel,
Just wondering if there is any progress on bringing the packages up to parity between the Managed Package and Standard runtimes?
The development/deployment process for OmniStudio is much smoother with the Standard runtime, so would be ideal to have orgs on this setting. Any news?
Cheers
Ben

**Comment by eschweitzer78** [comment]
Hello @batesbw Ben!

Salesforce just announced support for the ability to override standard runtime OmniStudio elements as part of the upcoming v252 (winter '25) release notes. This is generally speaking good news.

However, preview sandboxes have just been made available and there is currently very little documentation on the topic. Which means that it may take some time before we are able to assess the scope of work as we know there are differences between standard runtime and packaged runtime elements. And more time before we are able to reach parity.

But I'll certainly get back to you once we have the scope of work so that we can align and perhaps use your scripts as a further test bed if that's something your team is willing to entertain.

Stay tuned!
Emmanuel

**Reply by batesbw** [reply]
Great news Emmanuel, we would definitely be keen to help out with any testing - just reach out when you're at that stage and we will help however we can.
Cheers
Ben
